library/core/src/panicking.rs

```diff
@@ -84,62 +149,16 @@ fn panic_bounds_check(index: usize, len: usize) -> ! {
     panic!("index out of bounds: the len is {len} but the index is {index}")
 }
 
-// This function is called directly by the codegen backend, and must not have
-// any extra arguments (including those synthesized by track_caller).
+/// Panic because we cannot unwind out of a function.
+///
+/// This function is called directly by the codegen backend, and must not have
+/// any extra arguments (including those synthesized by track_caller).
 #[cold]
-#[inline(never)]
```

Comment:
Maybe re-adding this will work? Also please add the inline attributes of panic_str_nounwind back if that works.